components/style/custom_properties.rs

```diff
@@ -168,6 +172,38 @@ where
     }
 }
 
+/// An iterator for OrderedMap
+pub struct OrderedMapIterator<'a, K, V>
+where
+    K: 'a + Eq + Hash + Clone, V: 'a,
+{
+    /// The OrderedMap itself.
+    inner: &'a OrderedMap<K, V>,
+    /// The position of the iterator.
+    pos: usize,
+}
+
+impl<'a, K, V> Iterator for OrderedMapIterator<'a, K, V>
+where
+    K: Eq + Hash + Clone,
+{
+    type Item = (&'a K, &'a V);
+
+    fn next(&mut self) -> Option<Self::Item> {
+        let ref index = self.inner.index;
+        if self.pos >= index.len() {
+            return None;
+        }
+
+        let ref name = index[index.len() - self.pos - 1];
+        self.pos += 1;
+        match self.inner.values.get(name) {
```

Comment:
It should be an error if we get None from this, right?  Can we just unwrap() instead of returning None?